telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -18,16 +18,25 @@
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 """This module contains the ChatJoinRequestHandler class."""
 
+from typing import FrozenSet, Optional
 
 from telegram import Update
+from telegram._utils.defaultvalue import DEFAULT_TRUE
+from telegram._utils.types import RT, SCT, DVInput
 from telegram.ext._handler import BaseHandler
-from telegram.ext._utils.types import CCT
+from telegram.ext._utils.types import CCT, HandlerCallback
 
 
 class ChatJoinRequestHandler(BaseHandler[Update, CCT]):
     """BaseHandler class to handle Telegram updates that contain
     :attr:`telegram.Update.chat_join_request`.
 
+    Note:
+        If either the :paramref:`username` or the :paramref:`chat_id` corresponds to one
+        in the appropriate, the request will be passed to this handler.
```

Comment:
```suggestion
        If neither of :paramref:`username` and the :paramref:`chat_id` are passed, this handlers accepts *any* join request. Otherwise, this handler accepts all requests to join chats for which the chat ID is listed in :paramref:`chat_id` or the username is listed in :paramref:`username`, or both.
```

Making this note a bit more verbose should avoid some confusion :)